hack/check.sh

```diff
@@ -10,7 +10,7 @@ cd "${curDir}/../" || return
 check() {
     # gofmt
     echo "CHECK: gofmt, check code formats"
-    result=$(find . -name '*.go' -print0 | xargs -0 gofmt -s -l -d 2>/dev/null)
+    result=$(find . ! -path './vendor*' -name '*.go' -print0 | xargs -0 gofmt -s -l -d 2>/dev/null)
```

Comment:
Do you need to add `vendor` here? @SataQiu

Reply:
This is an implicit bug. It does not exclude possible `vendor` directory. If we run `make go-mod-vendor` and then run `make check`, it will failed.

Reply:
We should not check vendor, Right?

Reply:
Yes, but after migrating to golangci-lint we don't run `make go-mod-vendor` anymore. So I think you can remove this.